libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationTelemetry.java

```diff
@@ -169,6 +165,8 @@ void initialize(@NonNull Context context, @NonNull String accessToken,
 
       isInitialized = true;
     }
+    // Setup the listeners
+    initEventDispatcherListeners(navigation);
```

Comment:
@ericrwolfe the listeners are also added every time we init now instead of the first time.  Arrival events weren't firing after rotation because of this (listener would be null).